components/templates/SlideshowPage/index.tsx

```diff
@@ -0,0 +1,16 @@
+import { FunctionComponent } from "react";
+import { getSlideshowDataByUri } from "@/lib/api/slideshows";
+import SlideshowPageClient from "./client";
+
+const SlideshowPage: FunctionComponent<{
+  section: string;
+  data: PageEntry;
+  locale: string;
+  previewToken?: string;
+}> = async ({ data: { uri }, section, locale, previewToken }) => {
+  const data = await getSlideshowDataByUri(uri, locale, previewToken);
+
+  return <SlideshowPageClient {...{ data, section }} />;
```

Comment:
slideshows had their own custom data fetch, this has been moved into a server file for the template